src/main/java/gift/dto/wishlist/WishListResponse.java

```diff
@@ -12,4 +14,16 @@ public record WishListResponse(
    int totalPrice
 ) {
 
+    public static WishListResponse from(WishList entity){
+        return new WishListResponse(
+            entity.getMember().getId(),
+            entity.getMember().getEmail(),
+            entity.getProduct().getId(),
+            entity.getProduct().getName(),
+            entity.getProduct().getPrice(),
+            entity.getQuantity(),
+            entity.getQuantity() * entity.getProduct().getPrice()
+        );
+    }
```

Comment:
변수로 빼서 주입하는 건 어떨까요? 
```suggestion
    public static WishListResponse from(WishList entity){
        var member = entity.getMember();
        var product = entity.getProduct();
        return new WishListResponse(
            member.getId(),
            member.getEmail(),
            product.getId(),
            product.getName(),
            product.getPrice(),
            entity.getQuantity(),
            entity.getQuantity() * product.getPrice()
        );
    }
```

Reply:
```suggestion
    public static WishListResponse from(WishList entity){
        return new WishListResponse(
           entity.getProduct(),
           entity.getMember(),
           entity.getQuantity(),
        );
    }
```
또는 객체 자체를 넘겨줘도 될 것 같아요.

Reply:
피드백 감사합니다. 클라이언트에게 가는 Response에는 member의 id, password 같은 정보가 넘어가지 않았으면 합니다 ! 
그래서 제시해 주신 첫번째 코드처럼 변수로 빼서 주입하여 가독성을 개선하되, 객체 자체를 전달하지는 않는 방향으로 수정했습니다.